api/public.proto

```diff
@@ -874,6 +931,7 @@ message Trip {
     Stop.Reference destination = 4;
     optional Vehicle.Reference vehicle = 5;
     bool direction_id = 6;
+    optional Shape.Reference shape = 7;
```

Comment:
Do we need the Shape reference to be here? I understand this type already has a lot of stuff in it, but not sure the shape is important enough to be returned in reference types?

Reply:
I've removed this in the latest iteration. I realized shortly after I sent the PR that I never actually populated this field anywhere haha.

Reply:
Nice!